src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1137,4 +1137,83 @@ public static void forceJVMLocaleToUSEnglish() {
         final List<T> sorted = values.stream().sorted().collect(Collectors.toList());
         return sorted.get(sorted.size() / 2);
     }
+
+    /**
+     * Splits a String using indexOf instead of regex to speed things up.
+     * If given an empty delimiter, will return each character in the string as a token.
+     * @param str the string to split.
+     * @param delimiter the delimiter used to split the string.
+     * @return A {@link List} of {@link String} tokens.
+     */
+    public static List<String> split(final String str, final String delimiter) {
+        // This is 10 because the ArrayList default capacity is 10 (but private).
+        return split(str, delimiter, 10);
+    }
+
+    /**
+     * Splits a String using indexOf instead of regex to speed things up.
+     *
+     * @param str the string to split.
+     * @param delimiter the delimiter used to split the string.
+     * @return A {@link List} of {@link String} tokens.
+     */
+    public static List<String> split(final String str, final char delimiter) {
+        final List<String> tokens = ParsingUtils.split( str, delimiter );
+
+        // We must adjust for splitting on the last character so that the results here will be the same as
+        // the results of String.split:
+        if ( delimiter == str.charAt(str.length() - 1)) {
+            tokens.remove( tokens.size() - 1 );
```

Comment:
If you are splitting a single-character String containing only the delimiter (eg., the String `":"` with a delimiter of `':'`) this will (I believe) cause an empty tokens array to be returned. Does this match the behavior of `String.split()` with no limit argument?

Reply:
Added in the test case.